### ISSUE TITLE
compile: syntax error due to multiline comment

### ISSUE THREAD

**Issue by MrSaints** [graphql-cli/graphql-cli#134]
The output of `get-schema` (via `schema.graphql`) appears to include descriptions in multiline comment blocks. For example:

```
"""The ID of the object."""
id: ID!
```

Resulting in the following when `compile` is executed:

```
GraphQLError: Syntax Error GraphQL request (9:3) Expected Name, found String
```

I did not have this issue prior to upgrading. Any idea how I can resolve this? 

This happened after going from: `"graphql-cli": "2.0.4",` -> `"graphql-cli": "2.9.4",`

**Comment by MrSaints** [comment]
Whoops, sorry. Ignore this. I'm getting my tooling mixed up! This appears to be a problem with `relay-compiler`.

**Comment by pranaygp** [comment]
w.r.t relay, This has been fixed in `1.5.0-rc.1`, which uses the latest version of `graphql ` where the fix has been implemented. 

Please update to that version (or 1.5.0 once it's released) if you want this fixed
